packages/core/stories/stack-layout.doc.stories.mdx

```diff
@@ -12,22 +12,36 @@ import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
 
 # Stack layout
 
-Stack layout dictates a vertical order and direction for UI elements.
+Stack layout dictates a vertical or horizontal order and direction for UI elements.
 It builds from flex layout, the height is driven by the child elements, and there are visual separators that can be turned on/off (default) between items.
 
-It can be used for cases in which the direction needs to be consistently vertical, for example a list.
+It can be used with `column` direction for cases in which the direction needs to be consistently vertical, for example a list.
+It can be used with `row` direction to layout static components or information in a horizontal orientation, such as metric data displays, contact information or page filters.
 
 <Canvas>
   <Story id="core-layout-stacklayout--default-stack-layout" />
 </Canvas>
 
 ## Use Stack layout when...
 
-You need a layout that is responsive by default and will break into two columns as soon as there is not enough space.
+- You need a layout that is consistently horizontal or vertical.
+- You need your items to be either scrollable or you want to use breakpoints for responsiveness.
```

Comment:
Maybe worth re-wording slightly to mention toggling between row/column via the responsive direction prop?